src/VCluster/VCluster_base.hpp

```diff
@@ -69,7 +69,7 @@ extern size_t tot_recv;
 
 ///////////////////// Post functions /////////////
 
-extern size_t NBX_cnt;
+//extern size_t NBX_cnt;
```

Comment:
If it's removed here to become a class member, it has to be removed from VCluster.cpp, where the memory is actually allocated

Reply:
Done!